src/constants/colors.js

```diff
@@ -1,5 +1,5 @@
 export const WHITE = '#FFFFFF';
-export const BACKGROUND = '#F5FCFF';
+export const WHITE_BACKGROUND = '#F5FCFF';
```

Comment:
Why Rename this to WHITE_BACKGROUND. I like the name background. Is more repreentative. Tomorrow if you change the background color to red you need to also rename the variable to RED_BACKGROUND to stay consistent

Reply:
Hahaha that's something I commented. I prefer color name variables to be colors. If you need the background color to change just change de varaible that you use, not it's value. Thats my opinion.